app/controllers/guidances_controller.rb

```diff
@@ -67,8 +67,10 @@ def admin_update
     authorize guidance
     guidance.text = params["guidance-text"]
     
-    if guidance.update_attributes(guidance_params)
-
+    attrs = guidance_params
+    attrs[:theme_ids] = [] unless attrs[:theme_ids]
```

Comment:
Another solution could be including a hidden field theme_ids for when no themes are selected through JS associated to guidances view. We did like that for the notification preferences (e.g. https://github.com/DMPRoadmap/roadmap/blob/development/lib/assets/javascripts/views/users/notification_preferences.js#L15)